share/cmake/modules/FindExtPackages.cmake

```diff
@@ -52,18 +52,8 @@ if(OCIO_BUILD_APPS)
     find_package(lcms2 2.2 REQUIRED)
 endif()
 
-if(OCIO_BUILD_PYTHON)
-
-    # NOTE: Depending of the compiler version pybind11 2.4.3 does not compile 
-    # with C++17 so, if you change the pybind11 version update the code to 
-    # compile pybind11 and dependencies with C++17 or higher i.e. remove the 
-    # cap of C++ version in FindPybind11.cmake and 
-    # src/bindings/python/CMakeLists.txt.
-
-    # pybind11
-    # https://github.com/pybind/pybind11
-    find_package(pybind11 2.6.1 REQUIRED)
-endif()
+set (OCIO_PREFERRED_PYTHON_VERSION "" CACHE STRING
```

Comment:
Thinking we might want to make this an option declaration in the root `CMakeLists.txt` file for better visibility, which could then be referenced here.

Reply:
I can move it to the top level if you want. Also, do you prefer this name, or the less wordy `OCIO_PYTHON_VERSION`?

Reply:
Less wordy one `OCIO_PYTHON_VERSION` with a warning (or error) if the option is defined but `OCIO_BUILD_PYTHON` is OFF.

Reply:
done